devel/libcdio/Portfile

```diff
@@ -33,6 +33,12 @@ checksums           rmd160 6b0e4917cd2dd5cac0b7afdb70789bf0ad2aed79 \
 # doesn't understand the pramga usage in the Tiger headers - gcc bug 50909
 patchfiles          patch-libcdio-lib-iso9660-move-pragma.diff
 
+# Fix the crash reading CD on macos Ventura
+# Remove the unecessary addtional byte added to the TOC buffer
+if {${os.platform} eq "darwin" && ${os.major} >= 22} {
+    patchfiles      remove-additional-byte-TOC-buffer.diff
```

Comment:
sorry, one more change needed and then I'll merge. You will need to use "patchfiles-append", otherwise you will remove the patchfile added a few lines above.

Reply:
Indeed! Good catch!

Reply:
Oh crap I'm sorry. I edit right away.